firecracker/packer/build-rootfs.pkr.hcl

```diff
@@ -0,0 +1,119 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.0.6" # This is when `ami_org_arns` was added
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+variable "instance_type" {
+  description = "The type of instance used to *build* the image"
+  type        = string
+  # This is the value that Buildkite uses in their Packer template
+  default = "m5.xlarge"
+}
+
+variable "region" {
+  description = "The AWS region in which the base AMI is found *and* where the new AMI will be created"
+  type        = string
+  default     = "us-east-1"
+}
+
+variable "aws_profile" {
+  description = "The AWS connection profile to use when creating the image"
+  type        = string
+  default     = env("AWS_PROFILE")
+}
+
+variable "dist_folder" {
+  description = "Location of grapl_root/dist"
+  type        = string
+}
+
+variable "image_name" {
+  description = "The name of the artifact that will end up in dist/, excluding .tar.gz"
+  type        = string
+  default     = "firecracker_rootfs"
+}
+
+variable "debian_version" {
+  description = "Which version of Debian to use with debootstrap."
+  type        = string
+  default     = "bullseye"
```

Comment:
Just out of curiosity, is there a numerical version we could use instead? It's not a big deal either way, but I just personally hate the Debian names because there's no inherent ordering to them.